### ISSUE TITLE
Vue 3 Support

### ISSUE THREAD

**Issue by Cmacu** [dmtrbrl/vue-trend-chart#31]
Tried to use it with Vue 3. Getting the following error:
```
vue-trend-chart.esm.js?21e9:723 Uncaught (in promise) TypeError: h is not a function
    at Proxy.render (vue-trend-chart.esm.js?21e9:723)
    at renderComponentRoot (runtime-core.esm-bundler.js?9e79:846)
    at componentEffect (runtime-core.esm-bundler.js?9e79:4280)
    at reactiveEffect (reactivity.esm-bundler.js?0a2b:42)
    at effect (reactivity.esm-bundler.js?0a2b:17)
    at setupRenderEffect (runtime-core.esm-bundler.js?9e79:4263)
    at mountComponent (runtime-core.esm-bundler.js?9e79:4222)
    at processComponent (runtime-core.esm-bundler.js?9e79:4182)
    at patch (runtime-core.esm-bundler.js?9e79:3791)
    at mountChildren (runtime-core.esm-bundler.js?9e79:3975)
render @ vue-trend-chart.esm.js?21e9:723
```

**Comment by antpv** [comment]
Some problem..

**Comment by productdevbook** [comment]
please vue3 support add

**Comment by productdevbook** [comment]
I hope this resolves soon.

**Comment by dmtrbrl** [comment]
Fixed in v0.17.0
